### ISSUE TITLE
11-Set-up-reduxStore

### ISSUE THREAD

**Issue by Kmeiklejohn** [tander29/kwitter#18]
1. Create a global state
2. implement redux in components

**Comment by tander29** [comment]
I don't think we can call this done before we have implemented using redux to pass properties to components after fetching an initial state
